src/views/explore/explore.jsx

```diff
@@ -0,0 +1,148 @@
+var injectIntl = require('react-intl').injectIntl;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var React = require('react');
+var render = require('../../lib/render.jsx');
+
+var Api = require('../../mixins/api.jsx');
+
+var Page = require('../../components/page/www/page.jsx');
+var Box = require('../../components/box/box.jsx');
+var Tabs = require('../../components/tabs/tabs.jsx');
+var SubNavigation = require('../../components/subnavigation/subnavigation.jsx');
+var Carousel = require('../../components/carousel/carousel.jsx');
+var offset = 0;
```

Comment:
`offset`, `more`, and `tab` should be included in the React component's state (`getInitialState`) rather than in the global module.